contents/blog/the-posthog-array-1-30-0.md

```diff
@@ -0,0 +1,130 @@
+---
+date: 2021-11-17
+title: Array 1.30.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes"]
+featuredImage: ../images/blog/array/1-28-0.png
+excerpt: Excerpt here
+---
+
+
+PostHog 1.30.0 includes a description here.
+
+## PostHog 1.30.0 release notes
+
+> For a better experience and new features, remember to [update your PostHog instance](/docs/self-host/configure/upgrading-posthog).
+
+**Release highlights:**
+- [Fresh new look-and-feel](#fresh-new-look-and-feel)
+- [Correlation analysis](#correlation-analysis)
+- [Saved insights](#saved-insights)
+- [Fully revamped recordings](#fully-revamped-recordings)
+
+### Fresh new look-and-feel
+
+PostHog just got a lot fresher! We have a brand new UI and layout that had been on the works, codename `lemonade` (because it's fresh). It's not only a coat of paint, we've pushed a lot of improvements to overall navigation and performance, we call it turbo mode.
```

Comment:
```suggestion
PostHog just got a lot fresher! We have a brand new UI and layout that had been in the works, codenamed `lemonade` (because it's fresh). It's not only a new coat of paint - we've also pushed a lot of improvements to overall navigation and performance too. We call it turbo mode.
```